components/Chat/ChatMessage.tsx

```diff
@@ -19,7 +19,6 @@ export const ChatMessage: FC<Props> = memo(
   ({ message, messageIndex, onEditMessage }) => {
     const { t } = useTranslation('chat');
     const [isEditing, setIsEditing] = useState<boolean>(false);
-    const [isHovering, setIsHovering] = useState<boolean>(false);
```

Comment:
No longer need to watch hovering state., using "group-hover" tailwind visibility extension instead.